src/components/Book.scss

```diff
@@ -0,0 +1,138 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;600;700&family=Roboto+Slab:wght@300;400;500;700&display=swap");
+
+.book-container {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.left-col {
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
+  gap: 1rem;
+}
+
+.progress {
+  width: 4.25rem;
+  height: 4.25rem;
+  border: 5px solid #4386bf;
+  border-left: 5px solid #f3f3f3;
+  border-radius: 50%;
+  transform: rotate(45deg);
+}
+
+.percentage {
+  font-size: 2rem;
+}
+
+.percent-completed {
+  opacity: 0.5;
+  font-size: 0.8rem;
+}
+
+.center-col {
+  gap: 1.5em;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.column-divider {
+  align-self: center;
+  border: 1px solid #e8e8e8;
+  height: 4.375rem;
+  margin: 0 4rem;
+  width: 0.125rem;
+}
+
+.right-col {
+  flex: 0.5;
+  gap: 1.5em;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: center;
+}
+
+.chapters {
+  width: 100%;
+  opacity: 0.5;
+  font-size: 0.8rem;
+  text-transform: uppercase;
+  margin-left: 10%;
+}
+
+.completed-chapter {
+  width: 100%;
+  font-size: 1rem;
+  text-transform: capitalize;
+  margin-left: 10%;
+}
+
+.progress-btn {
+  background: #0290ff;
+  border-radius: 3px;
+  padding: 0.5rem 1rem;
+  border: none;
+  color: #fff;
+  font-size: 0.8rem;
+  letter-spacing: 1px;
+  margin-left: 10%;
+  white-space: nowrap;
+}
+
+.book-details {
+  display: flex;
+  flex-direction: column;
+  gap: 0.4rem;
+}
+
+.btn-list {
+  display: flex;
+  flex-grow: 1;
+  list-style-type: none;
+  gap: 15px;
+}
+
+button {
+  font-family: RobotoSlab, "Arial", sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  border: none;
+  cursor: pointer;
+  background-color: #fff;
+}
+
+.book-author {
+  font-family: RobotoSlab, "Arial", sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+  background-color: #fff;
+}
```

Comment:
- [ ] Again here consider using a reusable CSS class for the `background-color` for a DRY css code approach.

Reply:
**Hi @Munsa1  👋,**
Thank you so much 🙏  for reviewing my project.
Your nice and constructive feedback is appreciated 🙏.
I have ✔️ and make them fixed.

![thank-you-gif-22](https://github.com/ab-noori/bookstore-cms/assets/111179533/0848000e-52ce-440c-a114-6c7f859a0eaa)